src/main/java/edu/harvard/iq/dataverse/util/BundleUtil.java

```diff
@@ -174,4 +174,12 @@ public static Locale getDefaultLocale() {
 
         return new Locale("en");
     }
+
+    public static String getStringFromDefaultBundleEng(String key )  {
+        ResourceBundle bundle = getResourceBundle(defaultBundleFile, new Locale("en"));
+        if (bundle == null) {
+            return null;
+        }
+        return getStringFromBundleNoMissingCheck(key, null, bundle);
+    }
```

Comment:
With the update to add toolName (which is great!), I think this is no longer needed. Is that right?

Reply:
@qqmyers You are right. I just removed it.